rosetta-lang/src/main/java/com/regnosys/rosetta/types/SubtypeRelation.java

```diff
@@ -12,10 +12,22 @@
 import com.regnosys.rosetta.types.builtin.RNumberType;
 import com.regnosys.rosetta.types.builtin.RStringType;
 
+import static com.regnosys.rosetta.utils.MetaUtil.*;
+
 public class SubtypeRelation {
 	@Inject 
 	private RBuiltinTypeService builtins;
 	
+	public boolean isSubtypeOf(RMetaAnnotatedType t1, RMetaAnnotatedType t2) {
```

Comment:
Would add an "early check" here to return true if `t1` equals `t2`.

Reply:
done